JudoPayDotNetIntegrationTests/Configuration.cs

```diff
@@ -1,12 +1,16 @@
-﻿namespace JudoPayDotNetIntegrationTests
+﻿using System.Configuration;
+using JudoPayDotNet.Enums;
+
+namespace JudoPayDotNetIntegrationTests
 {
-    static class Configuration
+    public class Configuration
     {
-        public const string Judoid = "100915867";
```

Comment:
We need to use Live tokens when tests are running against Production (which is the default) and Sandbox tokens when tests are running against UAT, hence moving all the values into the app.config so they can be transformed.